mopro-ios/README.md

```diff
@@ -25,3 +25,27 @@ LIBRARY_SEARCH_PATHS=${SRCROOT}/../../Libs
 OTHER_LDFLAGS=-lmopro_ffi
 USER_HEADER_SEARCH_PATHS=${SRCROOT}/../../include
 ```
+
+## Tests
+
+There are two ways to run tests:
```

Comment:
Can we clarify that this is running tests for the example app? Which by proxy is testing the library

Reply:
updated